script.js

```diff
@@ -62,16 +125,21 @@ const inputAmount = document.getElementById("input-amount");
 
 const btn = document.getElementById("btn");
 
+// ADD NEW CURRENCY
+const addNewCurrencyButton = document.getElementById("add-currency");
+const inputNewCurrency = document.getElementById("new-currency-input");
+const inputNewCurrencyRate = document.getElementById("new-currency-rate-input");
+
 // GET FLAGS FROM SELECT
```

Comment:
I love the addition of the flags, it's a great detail! 🚀

⚔️ Challenge: What if I have other options besides USD, EUR, or CUB? What would happen if I had a list of larger flags? Could I match the input option with the corresponding flag?

Reply:
i will work on that!

Reply:
done